task_6/Anna.Kononenko/Task4_Symma_max_i_min_el.java

```diff
@@ -0,0 +1,23 @@
+package com;
+/**
+ 8. ���������� ����� ������������ � ������������� ��������� �������
+ */
+public class Task4_Symma_max_i_min_el {
+    public static void main(String[] args) {
+        int max = 0;
```

Comment:
You can't say max or min is equals 0, because it can be less than 0 number in array and no numbers 0 or greater and program won't work correctly. Usually they write max = array[0]; min = array[0]; this way we can be sure that we get the numbers only from the array.